worker/provisioner/provisioner_task.go

```diff
@@ -362,8 +362,25 @@ func (task *provisionerTask) processProfileChanges(ids []string) error {
 			}
 		} else if err != nil {
 			logger.Errorf("cannot upgrade machine's lxd profile: %s", err.Error())
+			// TODO lxd-profile 2018-11-14
+			// Remove call to SetUpgradeCharmProfileComplete
```

Comment:
For completeness I believe we should leave these calls in, so probably remove the TODO

Reply:
done